### DISCUSSION TITLE
@filamentScripts and Accessing Alpine via JS bundle

### DISCUSSION THREAD

**Discussion by hotsaucejake** [filamentphp/filament#7493] [Help] [answered]
Since Livewire 3 ships with AlpineJS by default, we need to remove it from our bundle.  

I do have this with a custom plugin that needs to be included in the bundle.  Livewire 3's documentation says to use the @livewireScriptConfig instead of @livewireScripts.

Filament replaces @livewireScripts with @filamentScripts.  How can I import the Livewire core ESM module into my bundle if @filamentScripts is used here?

Here is the documentation for reference: https://livewire.laravel.com/docs/upgrading#including-via-js-bundle

To be clear, I'm trying to make my app.js file like so (per the documentation):

```
import {Livewire, Alpine} from '../../vendor/livewire/livewire/dist/livewire.esm';
import AlpineFloatingUI from '@awcodes/alpine-floating-ui';

Alpine.plugin(AlpineFloatingUI);

Livewire.start();
```

**Comment by danharrin** [comment, marked as answer]
As it says in the upgrade guide, update Filament before updating Livewire. In which case, you should still have `@livewireScripts`.

You don't need to install `alpine-floating-ui`, we have that installed by default. Other plugins can be loaded via their CDN builds, with the Alpine initialisation event. Doesn't mean they need to be loaded on an actual CDN, just with that build file.

**Reply by hotsaucejake** [reply]
Thank you!  I did update Filament first and everything seems to be working.

Great work btw @danharrin !
